### PR TITLE
fix: hide the diagonses card if it's empty

### LINKED ISSUE #7187: User would be confused whether diagnoses are being loaded or does the consultation not have any diagnoses associated to it
**Describe the bug**
User would be confused whether diagnoses are being loaded or does the consultation not have any diagnoses associated to it.

**To Reproduce**

1. Open a patient that has no diagnoses

**Expected behavior**
It'd be nice to visibly show a text there are no diagnoses for this patient when no it is so.

**Screenshots**
image

### PR DESCRIPTION
**Pull request by aeswibon**
# Bug Fix

## Proposed Changes

- [x] Fixes #7187

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Screenshot

- Current staging:
image

- Fix:
image

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

File: src/Components/Facility/ConsultationDetails/index.tsx [hunk 1]
```diff
@@ -401,13 +401,16 @@ export const ConsultationDetails = (props: any) => {
...
         </div>
-        <div className="col-span-1 mt-2 overflow-hidden rounded-lg bg-white shadow">
-          <div className="px-4 py-2">
-            <DiagnosesListAccordion
-              diagnoses={consultationData.diagnoses ?? []}
-            />
-          </div>
-        </div>
+        {consultationData.diagnoses &&
+          consultationData.diagnoses.length > 0 && (
```
Comment (rithviknishad) [inline src/Components/Facility/ConsultationDetails/index.tsx]:
```suggestion
        {consultationData.diagnoses?.length && (
```

**Review by rithviknishad — COMMENTED** [review]
lgtm, just a minor nit pick. simplify the chekcs

**Review by rithviknishad — APPROVED** [review]

**Review by Ashesh3 — APPROVED** [review]

**Comment by nihal467** [comment]
@aeswibon 
image

when there is no diagnosis, there is a 0 being shown in the patient consultation page

File: src/Components/Facility/ConsultationDetails/index.tsx [hunk 2]
```diff
@@ -401,13 +401,15 @@ export const ConsultationDetails = (props: any) => {
             </div>
           </div>
         </div>
-        <div className="col-span-1 mt-2 overflow-hidden rounded-lg bg-white shadow">
-          <div className="px-4 py-2">
-            <DiagnosesListAccordion
-              diagnoses={consultationData.diagnoses ?? []}
-            />
+        {consultationData.diagnoses?.length && (
```
Comment (rithviknishad) [inline src/Components/Facility/ConsultationDetails/index.tsx]:
```suggestion
        {!!consultationData.diagnoses?.length && (
```

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
LGTM

**Comment by nihal467** [comment]
LGTM

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**